postgres/init.sls

```diff
@@ -126,6 +147,7 @@ postgresql-db-{{ name }}:
     - lc_ctype: {{ db.get('lc_ctype', 'en_US.UTF8') }}
     - lc_collate: {{ db.get('lc_collate', 'en_US.UTF8') }}
     - template: {{ db.get('template', 'template0') }}
+    - tablespace: {{ db.get('tablespace', 'pg_default') }}
```

Comment:
I didn't want to force the relation tablespace/database in any of the ways you mention because:

* In case you want to create a tablespace where lusers can create and/or manage databases at will, you don't have database/s to force a `require_in`, because you don't know them beforehand.

* Tablespaces can be created beforehand (ie, by some other tool/package you installed). In case you want to create databases in these tablespaces, you can just specify it in the database creation state, but I don't think it's wise to force a require to a tablespace state because you're not managing it directly.

Maybe I'm wrong? Perhaps is there a clean way to cover these two cases, that I'm missing?